scripts/unified-changelog.ts

```diff
@@ -56,16 +56,19 @@ const extractChangelogInfo = async (changelogPath: string): Promise<ChangelogInf
 
 const formatUnifiedChangelog = (infos: ChangelogInfo[]): string => {
   console.log('Formatting all changes into single changelog');
-  const formattedChangelog = infos
+  var formattedChangelogComponents = infos
```

Comment:
nit: I prefer `let` because I think block scoping is easier to reason about.

```suggestion
  let formattedChangelogComponents = infos
```